packages/storage-plus/src/de.rs

```diff
@@ -112,6 +112,22 @@ macro_rules! integer_de {
 
 integer_de!(for i8, u8, i16, u16, i32, u32, i64, u64, i128, u128);
 
+macro_rules! intkey_de {
```

Comment:
Ah, cool.. just started commenting how we need both. Then see how we do.

Nice!